DYDX-Bots/Insolvent_Contract/README.md

```diff
@@ -0,0 +1,31 @@
+# Insolvent Contract Bot
+
+## Description
+
+This bot detects when the contract is insolvent. Insolvency is refers to when `totalBorrowerDebtBalance` is greater than `totalActiveBalanceCurrentEpoch`. The checks for both variables in every block.
+
```

Comment:
` The checks for both variables in every block.` Do you mean `the bot checks ..`?